### DISCUSSION TITLE
“lab-specific” customization module

### DISCUSSION THREAD

**Discussion by ergonyc** [ndcn/omicser#187] [General]
Everyone!

We had discussed that the browser ought to include a space (module/tab/??) for embedding some specific functionalities needed for a type of -omic or for a particular labs workflow.    How can this be done?

Some key things to consider:
- UI format:  A tab, a pop-up, ??
- how to "modularize" it... the browser should work the same whether "lab specific" custom piece exists or not
- omic-specific functionality?  
-

**Comment by ddifraia** [comment]
@ergonyc having a look at the structure of the code, probably a tab-specific focus (es, proteome, metabolome etc...) depending on the type of data would be the easiest to implement probably.

**Reply by ricoderks** [reply]
@ergonyc @ddifraia I also think this is the best idea, probably in the playground tab.

**Reply by ddifraia** [reply]
Maybe an interactive one, that directly depends on the type of data loaded.

**Reply by ricoderks** [reply]
@ddifraia yes definitely! That would keep the GUI the simplest.

**Reply by ergonyc** [reply]
@ricoderks and @ddifraia I think that you should both impliment your specific tabs and share some of the decisions about how you integrate the loading of the module here.

**Reply by ddifraia** [reply]
@ergonyc @ricoderks  We are working with Max, on it. We still needs some time to get the real hands-on on the code. We will open a new discussion here when we have some initial results. But ideally we would need some "tag" in the curation process that will allow the identification of the different datasets. One could then read this tag and differentially create a new tab, depending on the type of experiment.

**Reply by ricoderks** [reply]
@ddifraia in the `db_config.yml` there is a tag called `omic_type`.
